server/config/config_test.go

```diff
@@ -188,6 +189,39 @@ address = "localhost:9090"
 	c.Assert(cfg.Metric.PushAddress, Equals, "localhost:9090")
 }
 
+func (s *testConfigSuite) TestMigrateFlags(c *C) {
+	load := func(s string) (*Config, error) {
+		cfg := NewConfig()
+		meta, err := toml.Decode(s, &cfg)
+		c.Assert(err, IsNil)
+		err = cfg.Adjust(&meta)
+		return cfg, err
+	}
+	cfg, err := load(`
```

Comment:
no well format

Reply:
em, what's your suggestion? The above test is using this format too.

Reply:
sorry, I thought it could be indented.